jbmc/regression/jbmc/core-models-required/test-no-package.desc

```diff
@@ -0,0 +1,9 @@
+KNOWNBUG
+HelloWorld.class
+--function "HelloWorld.helloWorld:()Ljava/lang/String;" --cp `../../../../scripts/format_classpath.sh . ../../../lib/java-models-library/target/core-models.jar` --context-include .
+^EXIT=0$
+^SIGNAL=0$
+--
+--
+`--context-include` with packageless terminate test-gen at `_Map_base::at`:
```

Comment:
Doesn't the same happen when running without `--cp ../../../../scripts/format_classpath.sh . ../../../lib/java-models-library/target/core-models.jar --context-include .` ? That would be a more general test case. Same for the other test case.

Reply:
Unfortunately, the cbmc branch does not recognise `--context-include` at this moment, so I cannot guarantee, but when I run test-gen on your branch without models-library, it does not cause invariant violation nor this mysterious interruption at `_Map_base::at` so I assume models library is needed to reproduce these bugs.

Reply:
@peterschrammel and I confirmed that `--context-include` contains wrong package, the issue is reproducible without the models path. However, `--context-include` points to a right package but not all necessary packages, test-gen (not JBMC) with models path causes invariant violation, which now is a separate PR https://github.com/diffblue/test-gen/pull/3616